tendrl/commons/jobs/__init__.py

```diff
@@ -94,7 +94,9 @@ def stop(self):
     def invoke_flow(self, flow_fqn, job):
         # flow_fqn eg:tendrl.node_agent.objects.abc.flows.temp_flows
         if "tendrl" in flow_fqn and "objects" in flow_fqn:
-            obj_name, flow_name = flow_fqn.split(".objects.")[-1].split(
+            obj_name = flow_fqn.split(".objects.")[-1].split(
+                ".flows.")[-2]
```

Comment:
Earlier we were trying to get obj_name and flow_name both in the same call. The value of obj_name used to be correct but flow name was not. Flow name used toe be reported as e.g. `create_volume.CreateVolume` whereas we need only `CreateVolume`, so we get these values separately now. Thats all...